### ISSUE TITLE
Category Sorting in Column View

### ISSUE THREAD

**Issue by GBVM** [jayelkaake/enhancedgrid#31]
This is not a real issue, but it would be nice to sort all categories by tree and not iD's when viewing the column for categories.

Is there a simple solution for this?

**Comment by pquerner** [comment]
You mean you want a category tree in the "category" column? Not simple just comma separated ids?

**Comment by GBVM** [comment]
@pquerner @jayelkaake 

Yes Pascal. The issue with sorting by ID is its hard to follow how the product is listed under each category/subcategory. By viewing this column in a listed tree format would make reference to the product listing much simpler.

EXAMPLE:

Current version - 	Engine Mechanical,PARTS,Oil System
(problem with this is ID's are not in order by Category Level)

Sorting by Tree Level - PARTS, Engine Mechanical, Oil System
or

                     PARTS
                     Engine Mechanical
                     Oil Systems

**Comment by YaegerDesign** [comment]
I would agree that this is a good improvement.
